src/mmw/mmw/settings/development.py

```diff
@@ -72,5 +72,17 @@
 
 # END TOOLBAR CONFIGURATION
 
+# UI CONFIGURATION
+
+DRAW_TOOLS = [
+    'SelectArea',   # Boundary Selector
```

Comment:
The keys were chosen to be identical to the Backbone View classes they control, but since in conversation we refer to these controls differently, those conversational names have been added as comments.